internal/backend/triggers/triggers.go

```diff
@@ -27,21 +30,71 @@ func (m *triggers) Create(ctx context.Context, trigger sdktypes.Trigger) (sdktyp
 	}
 
 	trigger = trigger.WithNewID()
+	if schedule, _ := trigger.Data()[sdktypes.ScheduleExpression].ToString(); schedule != "" {
```

Comment:
need to check against connection id, not by data

Reply:
done